merp_reservation_strategy/models/stock_quant.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 VentorTech OU
+# Part of Ventor modules. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class StockQuant(models.Model):
+    _inherit = 'stock.quant'
+
+    @api.model
+    def _update_reserved_quantity(self, product_id, location_id, quantity, lot_id=None, package_id=None, owner_id=None, strict=False):
+        """ Updates reserved quantity in quants
+        """
+        if self.env.user.company_id.custom_reservation_strategy:
+            self = self.with_context(merp_reservation_strategy='custom', merp_reservation_quantity=quantity)
+
+        return super(StockQuant, self)._update_reserved_quantity(product_id, location_id, quantity, lot_id, package_id, owner_id, strict)
+
+    def _gather(self, product_id, location_id, lot_id=None, package_id=None, owner_id=None, strict=False):
+        """ Gather (and reorder, if required) quants
+        """
```

Comment:
You need to add more extended comment about purpose of this redefined method. 
And why it is good place to redefine behavior here. 
So I, as side developer, will have no need to go and look at odoo code to understand why it is needed